docker-compose.backend.yml

```diff
@@ -13,6 +13,11 @@ services:
     volumes: # for data persistence
       - mongodb_vol:/data/db
 
+  redis:
+    image: redis
+    ports:
+      - '6379:6379'
+
```

Comment:
Include these like the other containers 

    restart: unless-stopped
    container_name: mongodb

![image](https://user-images.githubusercontent.com/77485021/200310720-a76d7365-fc98-4acb-a55b-b18bb54d807d.png)